hugo/content/blog/2024-05-19-mumble-1-5-634.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Mumble 1.5.634
+author: Hartmnt
+date: 2024-05-19
+categories:
+  - Release
+---
+
+After quite some time, we are proud to present you with the first stable release of the Mumble 1.5.x series. While this release may not be groundbreaking
+in terms of flashy and new features, we implemented a lot of quality-of-life and structural improvements to provide a solid foundation for the
+future development of the project.
+
+You can download the new version from our [**Downloads page**]({{< relref "/downloads" >}}) or the
+[**GitHub release page**](https://github.com/mumble-voip/mumble/releases/tag/v1.5.634) or from within your Windows client or software package
+management system.
+
+<!--more-->
+
+# Mumble 1.5.634
+
+Almost two years after the previous stable release Mumble 1.4.287, a lot of changes have been made to the codebase. With this update, you will surely
+notice some long-standing bugs fixed, as well as some convenient changes that were long overdue. In the time between the major versions, the project has
+exceeded the 10,000 commit mark and now has over 350 contributors. Thanks to all of you who contributed, translated, submitted issues, and tested
```

Comment:
```suggestion
exceeded the 10,000 commit mark and has now over 350 contributors. Thanks to all of you who contributed, translated, submitted issues, and tested
```

Reply:
I prefer *now has* over *has now*

I'm not sure *has now* is even correct grammar - it at least changes the meaning/mood a bit

Reply:
both are correct grammar, I prefer _now has_

Reply:
No problem.